src/main/play-doc/operation/ClusterStop.md

```diff
@@ -45,38 +45,26 @@ sudo service conductr-agent stop
 
 ## DC/OS mode
 
-> Do not Scale to 0, Suspend, or Destroy ConductR cluster directly from the Marathon UI without following the steps below.
- Doing so without following the steps below will cause the ConductR to be de-activated from DC/OS and forcefully detached from its still running tasks.
- These tasks will be orphaned and can't be terminated as it's no longer visible from DC/OS CLI's and Marathon's perspective.
+> Do not destroy the ConductR cluster directly from the UI without following the steps below.
 
-* Retrieve the service id of the ConductR service:
+* Stop the ConductR service via the DC/OS Services UI page:
 
-```bash
-dcos service
-NAME       HOST        ACTIVE  TASKS  CPU     MEM      DISK      ID
-conductr   10.0.1.118  True    0      11.8    42760.0  120804.0  my-conductr
 ```
-
-* Shutdown the ConductR service by using the `dcos service shutdown`. The command shuts down all ConductR executors on the Mesos agents
- and with it all tasks and bundles that have been created by ConductR. On Mesos master, the service is also marked as removed:
-
-```bash
-dcos service shutdown my-conductr
```

Comment:
This is consistently what caused the framework deleted problem. If you actually stick within the DC/OS UI and suspect tasks, the executors go away once the service is suspended after about two minutes. The service can then be destroyed and reinstalled with no problem and no name change necessary.